heft-plugins/heft-storybook-plugin/src/StorybookPlugin.ts

```diff
@@ -67,10 +73,8 @@ export interface IStorybookPluginOptions {
 }
 
 /** @public */
-export class StorybookPlugin implements IHeftPlugin<IStorybookPluginOptions> {
-  public readonly pluginName: string = PLUGIN_NAME;
-
-  private _logger!: ScopedLogger;
+export default class StorybookPlugin implements IHeftTaskPlugin<IStorybookPluginOptions> {
+  private _logger!: IScopedLogger;
   private _storykitPackageName!: string;
   private _startupModulePath!: string;
   private _resolvedStartupModulePath!: string;
```

Comment:
These non-null assertions kinda make me feel like the `options` object should be passed into the constructor, although that eliminates the possibility for a future stateless plugin.

Reply:
Honestly I think most of these options-based non-null assertions don't make much sense in the context of a Heft plugin. We should just be passing the values around that we get while invoking the tap. The goal should be to be as stateless as possible IMO.

I'll update it to load the values this way